src/utils.ts

```diff
@@ -1 +1,79 @@
+import { isUndefined } from 'lodash';
+
+import { NegatableOption, RequestSpec } from './RequestSpec';
+import { CmkQuery } from './types';
+import { requestSpecFromLegacy } from './webapi';
+
 export const titleCase = (str: string): string => str[0].toUpperCase() + str.slice(1).toLowerCase();
+
+export function contextEntryFromNegatableOption(option: NegatableOption, keyName: string) {
```

Comment:
please mention this new function in the commit message, or better: move it to a new commit.
did not expect code changes in this commit.